### ISSUE TITLE
CONFLICT_00213

### ISSUE THREAD

**Issue by kerrymcmahon** [kerrymcmahon/Baltimore-County-Stormwater#15]
Tree sites 030348UT and 030349UT were recently migrated into concept design. These are in direct conflict with MTI-076, a stormwater site that has already gone into a contract. 

The tree team must confirm immediately that these two tree sites have been placed on hold in the external process so that Kerry can update the database internally.

**Comment by kerrymcmahon** [comment]
The email was sent below and I also commented in the MCF. 

Dan,

The 20 BA County and 2 HA County recommended sites have been loaded. I did not mess too much with the “all sites”, because those listed as “removed” were already showing as removed in the database.

There are several conflicts with the new sites, hopefully Matt and Adam would have caught these anyways, but I figured I’d proactively point them out, since they are still training on the conflict resolution process. I need to ensure that these get updated on the tree side, I can update on the TMDL database side if confirmed.

030348UT - In direct conflict with a new SWM BMP that is already in a contract - Should be placed on hold until completion 

030349UT - In direct conflict with a new SWM BMP that is already in a contract- Should be placed on hold until completion

030362UT - Note CTP active project:

ISSUE_NAME  ISSUE_00013
GEN_COM - http://apps.roads.maryland.gov/WebProjectLifeCycle/ProjectInformation.aspx?projectno=BA458513

PROJECT_DESCRIPTION The project includes widening and resurfacing approximately one mile of the inner and outer loops of I-695 to provide continuous auxiliary lanes between MD 147 and MD 41.
RESEARCH_TYPE   Drainage Improvements

030365UT - In direct conflict with a new stormwater site already in concept design. - Should be placed on hold until completion

030379UT -  In direct conflict with a new stormwater site already in concept design. - Should be placed on hold until completion
